var/spack/repos/builtin/packages/bamdst/package.py

```diff
@@ -0,0 +1,47 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Bamdst(MakefilePackage):
+    "Bamdst is a a lightweight bam file depth statistical tool."
+
+    homepage = "https://github.com/shiquan/bamdst"
+    git      = "https://github.com/shiquan/bamdst.git"
+
+    version('master', git='https://github.com/shiquan/bamdst.git')
+
+    depends_on('zlib')
+
+    parallel = False
+
+    def edit(self, spec, prefix):
+        makefile = FileFilter('Makefile')
+        makefile.filter('CC= .*', 'CC = cc')
+
+    def install(self, spec, prefix):
+        make()
```

Comment:
No need for this line, it is already run in the `build` phase of `MakefilePackage`.

Reply:
Fixed.